index.html

```diff
@@ -73,8 +71,101 @@
       </div>
     </div>
   </header>
-  <main></main>
+  <main>
+    <section class="product">
+      <div class="carousel">
+        <div class="carousel__product">
+          <div class="carousel__img js-carousel-img">
+            <div class="carousel__item product-banner js-carousel-item" style="background-image: url('image/Banner.jpg')"></div>
+            <div class="carousel__item product-banner js-carousel-item" style="background-image: url('image/Banner.jpg')"></div>
+            <div class="carousel__item product-banner js-carousel-item" style="background-image: url('image/Banner.jpg')"></div>
+          </div>
+          <div class="indicators">
+            <button type="button" class="js-dot js-go-to-slide" data-slide="0"></button>
+            <button type="button" class="js-dot js-go-to-slide" data-slide="1"></button>
+            <button type="button" class="js-dot js-go-to-slide" data-slide="2"></button>
```

Comment:
El uso de una clase `js-` no implica que se use para aplicarle estilos
```suggestion
            <button type="button" class="dot js-dot js-go-to-slide" data-slide="0"></button>
            <button type="button" class="dot js-dot js-go-to-slide" data-slide="1"></button>
            <button type="button" class="dot js-dot js-go-to-slide" data-slide="2"></button>
```